Integrations/integration-OTRS.yml

```diff
@@ -0,0 +1,971 @@
+    - name: title
+      description: Ticket Title of the ticket to update
+    - name: state
+      auto: PREDEFINED
+      predefined:
+      - New
+      - ClosedSuccessful
+      - ClosedUnsuccessful
+      - Open
+      - PendingReminder
+      description: Ticket State of the ticket to update
+    - name: priority
+      auto: PREDEFINED
+      predefined:
+      - 1VeryLow
+      - 2Low
+      - 3Normal
+      - 4High
+      - 5VeryHigh
+      description: Priority of the ticket to update
+    - name: article_subject
+      description: Article Subject of the ticket to update
+    - name: article_body
+      description: Article Body of the ticket to update
+    - name: queue
+      description: Queue that the ticket to update is in
+    - name: type
+      description: Ticket Type of the ticket to update
+    - name: dynamic_fields
+      description: 'Dynamic fields to apply to the updated ticket, in the format:
+        field1=value1,field2=value2. For example: ProcessManagementProcessID=1,ProcessManagementActivityStatus=2'
+    - name: attachment
+      description: 'File entry ID of the file to add as an attachment to the updated
+        ticket in CSV format, e.g., 123@20,124@21 '
+    outputs:
+    - contextPath: OTRS.Ticket.Article.Subject
+      description: Ticket article subject
+      type: string
+    - contextPath: OTRS.Ticket.Article.Body
+      description: Ticket article body
+      type: string
+    - contextPath: OTRS.Ticket.ID
+      description: Ticket ID
+      type: string
+    - contextPath: OTRS.Ticket.Created
+      description: Ticket creation date
+      type: date
+    - contextPath: OTRS.Ticket.Priority
+      description: Ticket priority
+      type: string
+    - contextPath: OTRS.Ticket.Queue
+      description: Queue that the ticket is in
+      type: string
+    - contextPath: OTRS.Ticket.State
+      description: Ticket state
+      type: string
+    - contextPath: OTRS.Ticket.Title
+      description: Ticket title
+      type: string
+    - contextPath: OTRS.Ticket.Type
+      description: Ticket type
+      type: string
+    description: Update an OTRS ticket
+  - name: otrs-close-ticket
+    arguments:
+    - name: ticket_id
+      required: true
+      description: Ticket ID of the ticket to close
+    - name: article_subject
+      required: true
+      description: Article Subject of the ticket to close
+    - name: article_body
+      required: true
+      description: Article Body of the ticket to close
+    outputs:
+    - contextPath: OTRS.Ticket.ID
+      description: Ticket ID
+      type: string
+    - contextPath: OTRS.Ticket.State
+      description: Ticket state
+      type: string
+    - contextPath: OTRS.Ticket.Article.Subject
+      description: Ticket article subject
+      type: string
+    - contextPath: OTRS.Ticket.Article.Body
+      description: Ticket article body
+      type: string
+    description: Close an OTRS ticket
+  dockerimage: demisto/pyotrs
```

Comment:
please create folder in `docker` with Dockerfile and all the dependencies

Reply:
https://github.com/demisto/content/tree/master/DockerImageFiles/pyotrs